test/Datadog.Trace.ClrProfiler.IntegrationTests/TestHelper.cs

```diff
@@ -305,7 +305,18 @@ public void Dispose()
                 _process.StandardInput.Write("q");
                 _process.StandardInput.Flush();
 
-                if (!_process.WaitForExit(5000))
+                // wait for IIS Express to exit
+                // check every second for 20 seconds
+                int timeoutRemaining = 20000;
+                const int timeoutStep = 1000;
+
+                while (!_process.HasExited && timeoutRemaining > 0)
```

Comment:
What is the benefit of doing this over `_process.WaitForExit(20000)`?

Reply:
Ugh. My turn to say, "I'm just stupid." I'll revert this. For some reason I thought this would _always_ block for 20 seconds (which makes no sense) and I was trying to make this finish faster. Please ignore.